internal/bloblang/parser/mapping_parser_test.go

```diff
@@ -446,6 +474,25 @@ root = this.apply("foo")`,
 				Content: `{"applied":["bar","foo"],"foo":{"bar":{"outer":{"inner":"hello world"}},"static":"this is valid"}}`,
 			},
 		},
+		"test single root mapping": {
+			mapping: `"foo" == "bar"`,
+			input: []part{
+				{Content: ``},
+			},
+			output: part{
+				Content: `false`,
+			},
+		},
+		"test single root mapping with blobl shebang": {
+			mapping: `#!blobl
+"foo" == "bar"`,
+			input: []part{
+				{Content: ``},
+			},
+			output: part{
+				Content: `false`,
+			},
+		},
 		"test imported map": {
 			mapping: fmt.Sprintf(`import "%v"
```

Comment:
Should Benthos reject `mapping: import "foobar.blobl"`? Currently, it's equivalent to `noop`.

Reply:
Good question, probably as it's likely unintentional that someone would do this. I'd probably expose this as a lint warning rather than error which we dont have for bloblang right now. Maybe lets leave this for now.